docs/operational_guide/repairs.md

```diff
@@ -0,0 +1,34 @@
+# Background Repairs (beta)
+
+## Overview
```

Comment:
can you add a quick tl;dr for what background repairs are, what they do, and why they might be necessary for the uninitiated?